feat:configure frontend dependencies and folder structure

HurasAdam opened the pull request.

HurasAdam committed:
feat:configure frontend dependencies and folder structure

sovsky commented on postcss.config.js:
Czemu config frontendu jest poza katalogiem frontend ?

HurasAdam committed:
feat:
-Adjust config file placement
-Added Navbar component

HurasAdam committed:
feat:
-Added Hero section component.

kubaplayer05 committed:
feat:
- init shadcn/ui
- add basic components from shadcn/ui
- add pnpm-lock.yaml file for pnpm support
